src/components/NcUserStatusIcon/NcUserStatusIcon.vue

```diff
@@ -39,6 +50,7 @@ import { t } from '../../l10n.ts'
 
 import onlineSvg from '../../assets/status-icons/user-status-online.svg?raw'
 import awaySvg from '../../assets/status-icons/user-status-away.svg?raw'
+import busySvg from '../../assets/status-icons/user-status-busy.svg?raw'
 import dndSvg from '../../assets/status-icons/user-status-dnd.svg?raw'
 import invisibleSvg from '../../assets/status-icons/user-status-invisible.svg?raw'
```

Comment:
what I ask myself always here is why not just import from mdi?

Reply:
Fill color is currently forced inside of each svg.
We can do it inside of the component with mdi path, of course, but I already copied new svgs from Material symbols 👀

Reply:
> Fill color is currently forced inside of each svg

Yes its more like expected that to happen in `NcUserStatusIcon` - but no worries thats completely unrelated to your PR ;)